.dotnet/api/OpenAI.netstandard2.0.cs

```diff
@@ -1726,9 +1357,13 @@ public class FineTuningClient {
         protected internal FineTuningClient(ClientPipeline pipeline, Uri endpoint, OpenAIClientOptions options);
         public virtual ClientPipeline Pipeline { get; }
         public virtual ClientResult CancelJob(string jobId, RequestOptions options);
+        public ClientResult<FineTuningJob> CancelJob(string jobId, CancellationToken cancellationToken = default);
         public virtual Task<ClientResult> CancelJobAsync(string jobId, RequestOptions options);
+        public Task<ClientResult<FineTuningJob>> CancelJobAsync(string jobId, CancellationToken cancellationToken = default);
         public virtual ClientResult CreateJob(BinaryContent content, RequestOptions options = null);
+        public ClientResult<FineTuningJob> CreateJob(string training_file, string model, RequestOptions options = null);
```

Comment:
The pattern to follow for method parameters:

**Protocol** (should already be present, but here for contrast):
- Operation parameters that *aren't* part of the request body (path, query string, etc.) form the first parameters of the method
- If the operation accepts a request body, it also has a `BinaryContent` parameter
- All protocol methods then end with a `RequestOptions` parameter -- which should be optional if there's a BinaryContent parameter and required if there isn't (this addresses signature overlap)

**Convenience methods**:
- All required parameters, no matter what kind they are (query string, path, request body, etc.) are "promoted" into method parameters
- Optional parameters all go into an "*Options" class (like `ChatCompletionOptions`) that's the second-to-last, optional parameter
- The final parameter is `CancellationToken cancellationToken = default`

Convenience methods then just build off the protocol methods:
- Emplace any parameters that were "promoted" out of the request body back into the Options model instance
- Call the protocol method using the same path/query/etc. parameters and the output of `.ToBinaryContent()` from options for the body -- use `cancellationToken.ToRequestOptions()` for that parameter
- Deserialize the raw response from the protocol result into the desired output model and then use CilentResult.FromValue(...) for the convenience result

Applying that all here, a signature for `CreateJob` would be something like:
```csharp
public virtual ClientResult<FineTuningJob> CreateJob(string baseModelName, string trainingFileId, FineTuningJobOptions options = null, CancellationToken cancellationToken = default)
```

`FineTuningJobOptions` (or `FineTuningJobCreationOptions`) would be from the generated `CreateFineTuningJobRequest` type, just with the "promoted" `model` and `training_file` properties hidden (made internal).